curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -1501,7 +1508,7 @@ DataCache::DataCache(S3ClientAdaptorImpl *s3ClientAdaptor,
                      ChunkCacheManagerPtr chunkCacheManager, uint64_t chunkPos,
                      uint64_t len, const char *data)
     : s3ClientAdaptor_(s3ClientAdaptor), chunkCacheManager_(chunkCacheManager),
-      dirty_(true), delete_(false), inReadCache_(false) {
+      status_(DataCacheStatus::Dirty), inReadCache_(false) {
```

Comment:
why the initial status is Dirty?

Reply:
Because the data has been copied to the datacache during initialization